lib/Hero/Hero.js

```diff
@@ -0,0 +1,122 @@
+import React, { lazy, Suspense } from 'react';
+import PropTypes, { element, elementType, node } from 'prop-types';
+import { Button as ButtonComponent } from '../Button';
+import Content from './Content';
+const Carousel = lazy(() =>
+  import('../vendor/EmblaCarousel/Carousel/Carousel'),
+);
+
+const Hero = ({
+  classNames = {},
+  title,
```

Comment:
For non-carousel mode (single slide), let's not take separate `title`, `subTitle` and `imageUrl` props. Instead, let's keep it clean and have developers simply pass a 1-element array to the `slides` prop. If `slides.length === 1`, then show the 1-element hero. If `slides.length > 1`, then show the carousel.